docs/.vuepress/theme/styles/responsive.styl

```diff
@@ -6,10 +6,16 @@
   .page 
     width 100%
     max-width 1280px
+    padding-left cssGrad(320, 12, 1280, 96)
```

Comment:
Seems this function would be more useful if it didn't require the media query. Since you have to add the @media query anyway, is there any difference in functionality than having a `padding-left 12px` here?

Reply:
If so, I'd rather just put that vs. obscuring what's going on.

Reply:
I'd add that as a fallback, as the vw tag helps to keep things fluid (imo).
I have a much stronger preference for fluid sizing vs things being fixed and resizing at breakpoints

Reply:
Just added a fallback